src/domains/dashboard/components/head-main.tsx

```diff
@@ -0,0 +1,266 @@
+import { cond } from "ramda"
+import React from "react"
+import { ChartsMetadata } from "domains/global/types"
+import { ChartWrapper } from "domains/dashboard/components/chart-wrapper"
+
+interface Props {
+  charts: ChartsMetadata["charts"]
+  duration: number
+}
+export const HeadMain = ({
+  charts,
+  duration,
+}: Props) => {
+  const commonAttributes = {
+    after: -duration,
+    points: duration,
+  }
+
+  // todo
```

Comment:
So, what's the plan? We don't have this object in cloud, obviously =)

Reply:
well, you have this, please check again for `window.NETDATA.themes` :)
i've put it before, when i was adding charts (and you had it previously, too, when dashboard.js was there). It's of course `todo`, but it's working and it's making few things easier with static custom dashboards

https://github.com/netdata/dashboard/blob/f18b152b48ea18d687b3e3ddc2ebce28e52e718b/src/index-npm.ts#L10